common/src/main/java/com/wynntils/mc/event/ItemTooltipRenderEvent.java

```diff
@@ -1,32 +1,46 @@
 /*
- * Copyright © Wynntils 2022.
+ * Copyright © Wynntils 2022-2023.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
 package com.wynntils.mc.event;
 
 import com.mojang.blaze3d.vertex.PoseStack;
 import java.util.Collections;
 import java.util.List;
+import net.minecraft.client.gui.Font;
+import net.minecraft.client.gui.GuiGraphics;
 import net.minecraft.network.chat.Component;
 import net.minecraft.world.item.ItemStack;
 import net.minecraftforge.eventbus.api.Cancelable;
 import net.minecraftforge.eventbus.api.Event;
 
+/**
+ * This event is fired when an item tooltip is rendered.
+ *
+ * However, starting Minecraft 1.20, there is no longer a concerete way of rendering item tooltips.
```

Comment:
Typo in "concrete"

(Don't know why I can't make a suggestion... github moment)